File: 1_multiples_of_3_and_5/go/solve.go
package main

import "fmt"

func main() {
		solution := 0
		i := 1
    for i < 1000 {
        if i % 3 == 0 || i % 5 == 0 {
          solution = solution + i
        }
        i = i + 1
    }
    fmt.Println(solution)
}

File: 1_multiples_of_3_and_5/c/solve.c
#include <stdio.h>

int main(void)
{
  int solution = 0;

  for(int x = 0; x < 1000; x++) {
    if((x % 3 == 0) || (x % 5 == 0)) {
      solution = solution + x;
    };
  }

  printf("%d\n", solution);

  return 0;
}
